### ISSUE TITLE
hideConnectionUI does not hide the "Connect Wallet" button

### ISSUE THREAD

**Issue by paintoshi** [Uniswap/widgets#621]
**Bug Description**
According to the docs:

hideConnectionUI: Hide the widget's built-in wallet connection UI, including the connected account chip & 'Connect wallet to swap' button.

**Steps to Reproduce**
Provider in this case is a Web3Provider from wagmi, which should handle the wallet interaction.
```
<SwapWidget
  provider={provider}
  hideConnectionUI
/>
```

**Expected Behavior**
There should be no connection button. I still have it when my wallet is not connected.

![image](https://github.com/Uniswap/widgets/assets/88572775/8560048c-7f32-4d22-adf0-8ec366dec45d)

**Comment by N3koSempai** [comment]
yes. i have the same problem. but i think the problem is because the hideConnectionUi prop is not to hide the button. it is to hide the address that is shown on top of it.
